components/mpas-seaice/driver/ice_comp_mct.F

```diff
@@ -2544,6 +2544,7 @@ subroutine ice_export_mct(i2x_i, errorCode)   !{{{
             i2x_i % rAttr(index_i2x_Si_tref  ,n)  = atmosReferenceTemperature2m(i)
             i2x_i % rAttr(index_i2x_Si_qref  ,n)  = atmosReferenceHumidity2m(i)
             i2x_i % rAttr(index_i2x_Si_snowh ,n)  = snowVolumeCell(i) / ailohi
+            i2x_i % rAttr(index_i2x_Si_ithick,n)  = iceVolumeCell(i) / ailohi
```

Comment:
@akturner please review these additions to the sea ice coupler variables, which were added for Wave Watch III.